### PR TITLE
create PRs using ogr

### PR DESCRIPTION
**Pull request by marusinm**
- [x] replace creating PRs by Github API with ogr
- [x] labeling new pull request created by ogr (waiting for merging functionality into ogr)
- [x] implement [a fixed bug](https://github.com/packit-service/ogr/pull/130) in ogr

(No need for new tests since I replaced just bodies of functions which already have tests.)

### CODE REVIEW

**Comment by marusinm** [comment]
PR is finally ready for review by admins. @rpitonak

File: release_bot/github.py [hunk 1]
```diff
@@ -334,28 +312,22 @@ def make_pr(self, branch, version, log, changed_version_files, base='master', la
...
+        try:
+            new_pr = self.project.pr_create(
+                title=f'{version} release',
+                body=message,
+                target_branch=base,
+                source_branch=branch
+            )
+
+            self.logger.info(f"Created PR: {new_pr}")
+            if which_service(self.project) == "Github" and labels:
```
Comment (jpopelka) [inline release_bot/github.py]: just a tiny detail
```suggestion
            if labels and which_service(self.project) == "Github":
```

File: release_bot/github.py [hunk 2]
```diff
@@ -412,18 +387,16 @@ def pr_exists(self, name):
...
-            if not edges:
-                self.logger.debug(f"No open PR's found")
-                return False
-
-            for edge in reversed(edges):
-                cursor = edge['cursor']
-                match = re.match(name, edge['node']['title'].lower())
-                if match:
-                    return edge['node']['number']
+        merged_prs = self.walk_through_prs(PRStatus.open)
```
Comment (jpopelka) [inline release_bot/github.py]: I'm a bit confused here. Are you looking for merged or opened PRs?

```suggestion
        opened_prs = self.walk_through_prs(PRStatus.open)
```

File: release_bot/utils.py [hunk 3]
```diff
@@ -22,12 +22,41 @@
...
 
 logger = logging.getLogger('release-bot')
 
 
+def which_service(project):
+    """
+    Returns name one of the current git forges Github/Pagure/Gitlab
+
+    :param project: ogr-lib GitProject instance
+    :return: str
```
Comment (rpitonak) [inline release_bot/utils.py]: Why not enum?

**Pull request merged**